### PR TITLE
EL-1196: Add feature-flagged UI for downloading welsh CW forms

### PR DESCRIPTION
**Pull request by patrick-laa**
[Jira ticket](https://dsdmoj.atlassian.net/browse/EL-1196)

## What changed and why

- Add a feature flag. If it's not switched on, nothing changes
- If it's switched on, the user has to choose a language
- If they choose English, nothing changes
- If they choose Welsh, we record a different analytics event and try to populate a different PDF (but as it doesn't exist we currently raise an error)

## Checklist

Before you ask people to review this PR:

- Tests and rubocop should be passing
- Branch is generally up to date with main Github - definitely no conflicts
- No unnecessary whitespace changes. These make diffs harder to read and conflicts more likely.
- PR description says *what* changed and *why*, with a link to the JIRA story.
- Diff has been checked for unexpected changes being included.
- Commit messages say why the change was made.

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -503,6 +503,8 @@ en:
           attributes:
             form_type:
               blank: Select which form you need
+            language:
+              blank: Select which language you need the form in
```
Comment (MazOneTwoOne) [inline config/locales/en.yml:507]: At the moment, the validation error, doesn't display `Select which language you need the form in`, when no radio buttons are selected. 

Screenshot 2023-10-18 at 16 47 35



Also the validation error, doesn't disappear above the radio button when an option is selected. 

Screenshot 2023-10-18 at 16 48 28

**Review by MazOneTwoOne — CHANGES_REQUESTED** [review]

File: app/views/controlled_work_document_selections/new.html.slim [hunk 2]
```diff
@@ -4,17 +4,28 @@
   = link_to t("generic.back"), result_path(assessment_code: params[:assessment_code]), class: "govuk-back-link"
 
 .govuk-grid-column-two-thirds
-  = render "shared/heading"
+  = render "shared/heading", header_text: t(".title"), post_header_texts: t(".explainers")
+    - if @model.pensioner_disregard_applied?
+      = render "shared/banner",
+               heading: t("controlled_work_document_selections.new.banner.heading"),
+               body: t("controlled_work_document_selections.new.banner.body")
+
```
[the remark this answers is not in the document]
Comment (patrick-laa) [reply app/views/controlled_work_document_selections/new.html.slim:12]: We're not so much harmonising it as moving it. It used to be under the header when the header was the legend for the "form type" radio button set. In the new designs, the hint text is _above_ the new legend, but below the page title (now that it's no longer a 1-question page, the page title isn't the legend of the first question any more).

File: config/locales/en.yml [hunk 3]
```diff
@@ -2223,20 +2225,25 @@ en:
       - Payments for those who are not served by mains gas
   controlled_work_document_selections:
     new:
-      title: Which controlled work (CW) form do you need?
+      title: Download a controlled work (CW) form with your answers
```
[the remark this answers is not in the document]
Comment (patrick-laa) [reply config/locales/en.yml:2228]: Really good point! I'll ask on Jira.

File: config/locales/en.yml (hunk 1 above)
Comment (patrick-laa) [reply config/locales/en.yml:507]: My bad, a typo. Now fixed.

**Review by MazOneTwoOne — APPROVED** [review]

**Pull request merged**